leveldb/version_test.go

```diff
@@ -37,7 +37,11 @@ func TestVersionStaging(t *testing.T) {
 	tmp := make([]byte, 4)
 	mik := func(i uint64) []byte {
 		binary.BigEndian.PutUint32(tmp, uint32(i))
-		return []byte(makeInternalKey(nil, tmp, 0, keyTypeVal))
+		ik, err := makeInternalKey(nil, tmp, 0, keyTypeVal)
+		if err != nil {
+			panic(err) // Test helper - should not fail with valid parameters
+		}
```

Comment:
Same comment here and on lines 252-254 and 409-411 re using assertions instead of panics